ur_robot_driver/include/ur_robot_driver/ros/hardware_interface.h

```diff
@@ -157,6 +157,14 @@ class HardwareInterface : public hardware_interface::RobotHW
    */
   bool shouldResetControllers();
 
+  /*!
+   * \brief Mark next tick as logging tick
+   *
+   * \returns void
+   */
+  void shouldLogTemperature(bool value);
```

Comment:
I still think "publish" would be better than "log" here.

Reply:
Lets change that next time. I already tested on module and we have no automatic test for this.